project_0b/exercise_2_b_c_d_e.c

```diff
@@ -0,0 +1,83 @@
+/*
+Project 0b
+*/
+
+/*
+Exercise 2 b - Rewrite project_0a exercise 1, but allow to choose which scale to
+convert to and (Exercise 2 c) checks that the lowest temperature it can get to
+is -273.15C/-459.67F. (exercise 2 d) Keep asking for a new value to convert at
+the end of converting until (exercise 2 e) the user types "E" to stop the
+program.
+
+F = C × 1.8 + 32
+C = (32 - F)/1.8
+*/
+
+#include <ctype.h>
+#include <stdio.h>
+
+int temperature_converter(double temperature, char scale) {
+  double converted_temp;
+
+  double min_c = -273.15;
```

Comment:
This type of _universal_ constant should be declared in a global scope (and be const). The name should also be a bit more clear (at least `celsius` instead of `c`).